tests/css-hyphens/EPUB/package.opf

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<package xmlns="http://www.idpf.org/2007/opf" xmlns:epub="http://www.idpf.org/2007/ops" version="3.0" xml:lang="en" unique-identifier="pub-id">
+  <metadata xmlns:dc="http://purl.org/dc/elements/1.1/">
+    <dc:coverage>Prefixed CSS Properties</dc:coverage>
+    <dc:creator>Wendy Reid</dc:creator>
+    <dc:date>2022-03-02</dc:date>
+    <dc:description>Tests for the prefixed CSS property -epub-hyphens</dc:description>
+    <dc:identifier id="pub-id">css-hyphens</dc:identifier>
```

Comment:
I'll pull @dauwhe in here to correct me, but I think these are the only CSS tests we have, with possibly the exception for the MUST in the CSS section in RS regarding fonts. Since these are EPUB-specific CSS properties, I did suggest maybe epub-* but that didn't fit nicely into the three-letter opening scheme :). Adding prefix- feels long, maybe css-epub-*?

Reply:
I am also fine with css-epub-*, though the usual approach is to use the section headings. I would like to hear of @dlazin who is the guardian of naming 😀